Online/Menus/TitleMenu/AllowSwapToSameSubmenu.asm

```diff
@@ -10,8 +10,8 @@ bne CLEAR
 
 # Check force bool
 lbz r3, OFST_R13_FORCE_MENU_CLEAR(r13)
-cmpwi r3, 0
-bne CLEAR
+cmpwi r3, 1
```

Comment:
Do we know what this value is for? Maybe we should use a different r13 offset for `FORCE_MENU_CLEAR` instead?

Reply:
Its not clear to what id being used for exactly. 

I just know that after you leave tournament mode it is not reset.

But it is always updated when you open up tournament mode. Its safe to use it for now.